.github/workflows/pr_info_check.yml

```diff
@@ -0,0 +1,83 @@
+# NM was here
+
+name: 'PR Info Check'
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Check PR JIRA reference
```

Comment:
seems we don't need this since we don't have jira

Reply:
We have kind of jira. Maybe rename it to "issue reference", for example

Reply:
Renamed to "task"